app/forms/sufia/forms/admin_set_form.rb

```diff
@@ -3,6 +3,7 @@ module Forms
     class AdminSetForm < CurationConcerns::Forms::CollectionEditForm
       self.model_class = AdminSet
       self.terms = [:title, :description, :thumbnail_id]
+      attr_reader :workflow_name
```

Comment:
Why is this necessary? Can you delegate workflow_name to the permission_template instead?